package.sh

```diff
@@ -316,6 +316,13 @@ if [ $? -ne 0 ]; then
 fi
 echo "Debian package created successfully."
 
+fpm -s dir -t tar --description "$DESCRIPTION" $COMMON_FPM_ARGS && gzip influxdb.tar && mv influxdb.tar.gz influxdb_${VERSION}_${ARCH}.tar.gz
```

Comment:
This will create a tar archive that when decompressed, creates `etc/` and `opt/` in the same directory as the archive. I think it would be better if it decompressed into called `influxdb_version_arch/`, and *inside there* was `etc/` and `opt`. Make sense? To do this, perhaps it should just use `tar`.

I was about to merge this, but the behaviour above made me hesitate.